### PR TITLE
repository pattern to support play

### PR DESCRIPTION
**Pull request by otaiwo70**
repository pattern to support play

### CODE REVIEW

File: backend/game/irepository/irepository_player.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+class IRepositoryPlayer:
+    def __init__(self):
+        self.user = None
+
+    def save_player_db(self, player: Player):
+        self.user = User(monster_name=player.monster_name,
+                         username=player.username,
+                         password=player.password,
```
Comment (cvennel) [inline backend/game/irepository/irepository_player.py]: Don't think passwords should be save plaintext see review on #80

Comment (otaiwo70) [reply backend/game/irepository/irepository_player.py]: rectified by using password=md5.encrypt(player.password),

Comment (otaiwo70) [reply backend/game/irepository/irepository_player.py]: password reference removed

Comment (kaubster) [reply backend/game/irepository/irepository_player.py]: will have to deal with passwords eventually... but agree with removing for the moment.

**Pull request merged**